src/tasks/RuntimeConfigParser/RuntimeConfigParser.cs

```diff
@@ -109,7 +122,7 @@ private void CheckDuplicateProperties(IReadOnlyDictionary<string, string> proper
         {
             if (properties.ContainsKey(key.ItemSpec))
             {
-                throw new ArgumentException($"Property '{key}' can't be set by the user!");
+                Log.LogError($"Property '{key}' can't be set by the user!");
```

Comment:
This method will need to return bool too, if this is being changed to a LogError.

Reply:
Are you suggesting that I should return on this line? Like `return Log.LogError($"Property '{key}' can't be set by the user!");`

Reply:
no, I mean, after the LogError line, assuming that we want the task to fail at this point.